<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionInfoDialog</class>
 <widget class="QDialog" name="VersionInfoDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>260</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Meiryo UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Version Info</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="logo">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../rc.qrc">:/splash/src/ui/resource/splash/FboolDesktop_logo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentVersionInfo">
     <property name="font">
      <font>
       <family>Yu Gothic</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Version: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="copyright">
     <property name="font">
      <font>
       <family>Yu Gothic</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Copyright 2018 smartDIYs.co.,Ltd. All rights reserved.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="middleFrame">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CheckForUpdates">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:1px solid #CCCCCC;
	border-radius:3px;
	color:black;
}
QPushButton:hover
{
	background-color:#E6E6E6;
	border:1px solid #bfbfbf;
	border-radius:3px;
	color:black;
}
QPushButton:pressed
{
	background-color:#DEDEDE;
	border:1px solid #bfbfbf;
	border-radius:3px;
	color:black;
}</string>
        </property>
        <property name="text">
         <string>Check for Updates</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottomFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CloseButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Gothic</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	border:1px solid #CCCCCC;
	border-radius:3px;
	color:black;
}
QPushButton:hover
{
	background-color:#E6E6E6;
	border:1px solid #bfbfbf;
	border-radius:3px;
	color:black;
}
QPushButton:pressed
{
	background-color:#DEDEDE;
	border:1px solid #bfbfbf;
	border-radius:3px;
	color:black;
}</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../rc.qrc"/>
 </resources>
 <connections/>
</ui>
